enterprise/cmd/repo-updater/internal/authz/perms_syncer.go

```diff
@@ -440,16 +440,16 @@ func (s *PermsSyncer) syncUserPerms(ctx context.Context, userID int32, noPerms b
 
 	// Exclusions are relative to inclusions, so if there is no inclusion, exclusion
 	// are meaningless and no need to trigger a DB query.
-	if len(includePrefixSpecs) > 0 {
+	if len(includeContainsSpecs) > 0 {
 		rs, err := s.reposStore.RepoStore.ListRepoNames(ctx,
 			database.ReposListOptions{
-				ExternalRepoIncludePrefixes: includePrefixSpecs,
-				ExternalRepoExcludePrefixes: excludePrefixSpecs,
+				ExternalRepoIncludeContains: includeContainsSpecs,
+				ExternalRepoExcludeContains: excludeContainsSpecs,
 				OnlyPrivate:                 true,
 			},
 		)
 		if err != nil {
-			return errors.Wrap(err, "list external repositories by prefix matching")
+			return errors.Wrap(err, "list external repositories by depot matching")
```

Comment:
```suggestion
			return errors.Wrap(err, "list external repositories by contains matching")
```

There is no concept of depot at this layer.